cves/CVE-2009-1195.yml

```diff
@@ -193,8 +209,11 @@ lessons:
     applies:
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: |
+      The default functionality was not the most secure, which lead to the
+      vulnerability being there in the first palce, if no exec was the default
+      behavior this would have likely been avoided.
   yagni:
     applies:
     note:
```

Comment:
Could the execution of privilege escalating code be an example of a complex input?

Reply:
No, it was just a simple configuration flag, the language used for the flag is weird, but I wouldn't say the input was "complex"